runtime/server/server.go

```diff
@@ -191,6 +191,16 @@ func (s *Server) HTTPHandler(ctx context.Context, registerAdditionalHandlers fun
 		panic(err)
 	}
 
+	// custom APIs
+	err = gwMux.HandlePath("GET", "/v1/instances/{instance_id}/api/{name=**}", auth.GatewayMiddleware(s.aud, s.APIForName))
+	if err != nil {
+		panic(err)
+	}
+	err = gwMux.HandlePath("POST", "/v1/instances/{instance_id}/api/{name=**}", auth.GatewayMiddleware(s.aud, s.APIForName))
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Would suggest registering as a normal HTTP handler directly on `httpMux` (more portable/native).

As of Go 1.22, it supports path params as well (see the runtime proxy PR for an example).